assets/src/styles/blocks/Counter.scss

```diff
@@ -23,6 +23,9 @@
     }
 
     .progress-container {
+      _-- {
```

Comment:
Even though the style is called "progress bar in en form", the climate theme hides the progress bar. The result is somewhat non-sensical, and ideally I suppose the block should allow to have only text and still be put in the ENForm. But I'm keeping it to this quick fix, because we probably want to more broadly reconsider the current behavior of the counter block traveling into the ENForm block on page load. Instead these could use inner blocks in the new version.